assets/RNAseq_report.Rmd

```diff
@@ -1697,7 +1695,6 @@ The table below provides more detail on all enriched pathways."))
         q_list <- append(q_list, list(q))
```

Comment:
Regarding the pathway analysis part of the report: in the previous pipeline version, the pathway analysis was performed twice, once for generating the static plots inside the pathway analysis R script and once inside the Rmarkdown report to show the interactive plots. I think this is not required anymore, and just calling it once should suffice to produce both the interactive and static plots within the report, is that right?

Reply:
Only the function `gostplot` should be called twice, once with `interactive=T` and once with `interactive=F`.